src/web/routes/application/common/formatters/errors.js

```diff
@@ -0,0 +1,20 @@
+const mergeErrorMessage = (message, cause) => cause ? `${message}. ${cause.toString()}` : message
+
+const mergeErrorStack = (error, cause) => cause ? `${error.stack}\nCaused by: ${cause.stack}` : error.stack
+
+const wrapError = ({ cause, message, statusCode }) => {
+  const error = new Error()
```

Comment:
Didn't you say this would cause the error stack to start here and lose the context of where the error was first caught?

Reply:
This line now deals with that issue:
```
Error.captureStackTrace(error, mergeErrorProps)
```
see https://nodejs.org/api/errors.html#errors_error_capturestacktrace_targetobject_constructoropt